### ISSUE TITLE
Bugs:Youtube new patch bugs

### ISSUE THREAD

**Issue by Adity1243** [YT-Advanced/YT-Advanced#135]
1. Increase Hight of seekbar not working.
2. Hide video quality preference layout footer not working.
3. Video not geeting in full screen even after full screen option enabled.
4. playlist section gets vanish after we play video from history, close the video and go to seetings.

**Comment by YT-Advanced** [comment]
1. Confirm
2. Confirm
3. Cannot reproduce
4. Cannot reproduce (Can u give a video)
5. I think it should be normal

**Comment by Adity1243** [comment]
About 3rd issue i think its a/b testing bcus when i first opened the video quality seeting i saw additional seeting option there i found stats for nerds,vr and loop video and that not hidden.

**Comment by Adity1243** [comment]
4th issue similar like this issue i explained
 https://github.com/YT-Advanced/YT-Advanced/issues/132#issue-1888774386

**Comment by Adity1243** [comment]
4th issue is similar like this issue i explained in this thread :-
https://github.com/YT-Advanced/YT-Advanced/issues/132#issuecomment-1712746022

**Comment by Adity1243** [comment]
5th issue didnt occuring before patches and also i save some videos to watch later in liked videos section so its important to me to get fixed.

**Comment by Adity1243** [comment]
4th issue recording :-https://drive.google.com/file/d/1AoLQuE65FNnM5YgiOKxie3w8EtAnxCz6/view?usp=drivesdk

**Comment by YT-Advanced** [comment]
for 4th issues, I watched your video and all are normally 🤔
And 5th, can you exclude all patch related to playlist then try again ? I need to sure that it is not Youtube bug itself

**Comment by Adity1243** [comment]
4th issue i manually filled the screen with video but with that i can able to fill screen auto with video that is difference its not working now.
5th issue i tried to reproduce the issue in official youtube but it didnt happend so its not youtube bug.

**Comment by YT-Advanced** [comment]
No, I mean 5th ones you exclude all patch related to playlist (Only patch the other ones) and check until the issues solved, then send me which patch caused the error

**Comment by Adity1243** [comment]
I tried by excluding playlist patches that i know like:- hide mix playlist,Hide handle, but its specific that this bug occur only when we import our previous seeting not occuring  when we didnt import seetings.

**Comment by YT-Advanced** [comment]
So mannual toggle worked ?

**Comment by Adity1243** [comment]
Yes, but can you tell me which settings affect the playlist section, bcus i dont know.

**Comment by YT-Advanced** [comment]
> Yes, but can you tell me which settings affect the playlist section, bcus i dont know.so i can test it.

For easier checking, you can include only minimal patch, like only include MicroG and Hide Video Ads patch

**Comment by Adity1243** [comment]
Nope, hide video quality footer is still not hiding after patch with latest.

**Comment by YT-Advanced** [comment]
> Nope, hide video quality footer is still not hiding after patch with latest.

It seem like not hiding with some certain device (only for Caption). Now I don't have device to reproduce

**Comment by Adity1243** [comment]
Its just not hiding outside of video quality selection where auto and data save options re available
